### ISSUE TITLE
How to install the package?

### ISSUE THREAD

**Issue by wellyshen** [satya164/react-navigation-addons#8]
Hi @satya164 , Thank you for this repo, I can install this package through yarn but the module can't work properly, could you please check it?

**Comment by DavideValdo** [comment]
+1
Why not just relasing a yarn package?

**Comment by slorber** [comment]
@satya164 I can confirm the issue. The published package does not contain any index.js so you may have forgotten to build it before publish

**Comment by slorber** [comment]
![image](https://user-images.githubusercontent.com/749374/28266874-a6c40d0a-6af7-11e7-9670-f32bb75b209e.png)

**Comment by slorber** [comment]
I published a fork with the source files, but was unable to run the code correctly. I guess it's due to internal changes in react-navigation:

```
Cannot read property 'forEach' of undefined
    at _class._this._updateNavigationProps (withCachedChildNavigation.js:45)
    at _class.componentWillMount (withCachedChildNavigation.js:27)
```

If you want to try:

    "react-navigation-addons-seb": "^0.0.1",

**Comment by Philip-Nunoo** [comment]
https://github.com/satya164/react-navigation-addons/issues/13

**Comment by slorber** [comment]
@Philip-Nunoo does it work once installed?

**Comment by Philip-Nunoo** [comment]
@slorber yes it does. But you'd face a bit of challenge when you do have a TabNavigator nested within a StackNavigator.(That's for my case) But was able to figure it out. If anyone raises the issue of the tabNavigator not working I can go ahead and write my solution onto that issue. 

@wellyshen you can close this issue as it has been solved in #13

**Comment by wellyshen** [comment]
@Philip-Nunoo Sure, thanks a lot :)
